pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+## Problem
+- *Description of why you made this PR*
+
+## Solution
+- *And how do you fix that problem*
+
+## PR Status
+- [ ] Code finished.
+- [ ] Tested with Package_check.
+- [ ] Fix or enhancement tested.
+- [ ] Upgrade from last version tested.
+- [ ] Can be reviewed and tested.
+
+## Validation
+---
+- [ ] **Code review** : 
+- [ ] **Approval (LGTM)** :  
+
+- **CI succeeded** : 
+[![Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/pluxml_ynh%20PR-NUM-/badge/icon)](https://ci-apps-hq.yunohost.org/jenkins/job/pluxml_ynh%20PR-NUM-/)  
```

Comment:
```suggestion
[![Build Status](https://ci-apps-dev.yunohost.org/jenkins/job/pluxml_ynh%20PR-NUM-/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/pluxml_ynh%20PR-NUM-/)  
```

`high_quality` apps have a system to trigger a package_check automatically on PR, but if you don't ask for this tag, you must launch the package_check by yourself on https://ci-apps-dev.yunohost.org or on your side